### PR TITLE
Guardian Dream "Moisturizer"

### PR DESCRIPTION
**Pull request by prndev**

### CODE REVIEW

File: src/javascript/dreamsList.js [hunk 1]
```diff
@@ -202,6 +202,22 @@ window.dreamsGuardian={
...
...rizer is empty.”@@ she says as she turns to you.\n\nFor some reason, you are naked. $guardian bends down to examine your ball-sack.\n\n@@.guardiandream;“But these are nice and full.”@@ she comments while fondling your scrotum. @@.guardiandream;“When did you last cum?”@@\n\n“<<print window.timeCode.minutesToStr(playerCode.masturbate.sinceLastCum())>>” you answer truthfully.\n\n@@.guardiandream;“<<if playerCode.masturbate.sinceLastCum() > 2880>>It works best when they are full and ripe.<<else>>That will have to do.<<endif>>”@@\n\n$guardian caresses your caged cock with one hand while positioing the other below. Then she squeezes the cage twice. Each time, you can feel your penis pulse and you spurt a white substance into her hand. On the cupboard, you notice the bottle. Its curved dispenser looks very similar to your chastity device.\n\n$guardian proceeds to apply your personally delivered moisurizer to her face and hands. Then she gives you the empty bottle and a dildo.\n\n@@.guardiandream;“Sweetie, please fill up the bottle for me while I am away, will you?”@@\n\nShe kisses you on the forehead and leaves. Your penis twitches in anticipation, looking forward to the anal orgasms.\n",
```
Comment (MisterX421138) [inline src/javascript/dreamsList.js]:
```suggestion
		description: "$guardian is in front of the bathroom mirror, getting ready for the day. The fiddles around with a bottle, pushing the dispenser, but the bottle is empty.\n\n@@.guardiandream;“The moisturizer is empty.”@@ she says as she turns to you.\n\nYou suddenly realize that, for some reason, you are naked. $guardian bends down to examine your ball-sack.\n\n@@.guardiandream;“But this are nice and full.”@@ she comments while massaging your balls. @@.guardiandream;“When did you last cum?”@@\n\n“<<print window.timeCode.minutesToStr(playerCode.masturbate.sinceLastCum())>>” you answer truthfully.\n\n@@.guardiandream;“<<if playerCode.masturbate.sinceLastCum() > 2880>>Good. It works best when they are full and ripe.<<else>>That will have to do.<<endif>>”@@\n\n$guardian caresses your caged cock with one hand while positioing the other below, then she squeezes your cage twice. Each time, you can feel your penis pulse, and you spurt a white substance into her hand. Then, $guardian proceeds to apply your personally delivered moisurizer to her face and hands.\n\nIn the cupboard, you notice a strange bottle; its curved dispenser looks very similar to your chastity device. Guardian finishes moistuerizing and then gives you the empty bottle and a dildo.\n\n@@.guardiandream;“Sweetie, please fill up the bottle for me while I am away, will you?”@@\n\nShe kisses you on the forehead and leaves. Your penis twitches in anticipation, looking forward to the anal orgasms.\n",
```

Comment (MisterX421138) [reply src/javascript/dreamsList.js]: Some text editing to the dream

File: src/Twee/HomeRoutine/Good morning.twee [hunk 2]
```diff
@@ -620,12 +620,21 @@ Noticing your worried face at her angry outburst, she quickly reassures you.
...
-			
-			Finally, it stops, allowing you to withdraw the plug and put it on the night stand. You suddenly realize that you willingly endured the whole thing without touching yourself. You release the pillow and find that you have practically soaked the pillowcase, but your penis is limp despite how incredibly horny you are. You are not satisfied at all and can't even tell what pushed you to do that in the first place, but there is something compelling about surrendering to this longing for inescapable stimulation without release.
-			
-			An arousing idea comes to mind... if you managed to eventually enjoy this torture while in chastity, you might be able to when you're unlocked also. You decide to not touch yourself in the morning, no matter what.
-			
+			<<if kinkAllow.bdsm>>\
+				Your penis is free and with a butt-plug vibrating away on your prostate, it is hard not to start masturbating. Having been teased throughout the night, you come quickly, cum splattering all over your midriff and the sheets.
```
Comment (MisterX421138) [inline src/Twee/HomeRoutine/Good morning.twee]:
```suggestion
				Your penis is free and the butt-plug is vibrating away on your prostate, and you can't help but start masturbating. Having been teased throughout the night, you orgasm quickly, cum splattering all over your midriff and the sheets.
```

File: src/Twee/HomeRoutine/Good morning.twee [hunk 3]
```diff
@@ -620,12 +620,21 @@ Noticing your worried face at her angry outburst, she quickly reassures you.
...
+				
+				In your post-orgasmic daze, you realize you made a terrible mistake: The automatic vibrator has not finished "waking" you up, yet. What had been a pleasurable buzz on your insides a few seconds ago, now quickly becomes a source of increasing discomfort.
```
Comment (MisterX421138) [inline src/Twee/HomeRoutine/Good morning.twee]:
```suggestion
				In your post-orgasmic haze, you realize that you made a terrible mistake: the automatic vibrator has not finished "waking" you up, yet. What had been a pleasurable buzz on your insides a few seconds ago, quickly becomes a source of increasing discomfort.
```

File: src/Twee/HomeRoutine/Good morning.twee [hunk 4]
```diff
@@ -620,12 +620,21 @@ Noticing your worried face at her angry outburst, she quickly reassures you.
...
+				
+				@@.hero;I should not have touched myself.@@ you groan to yourself.
```
Comment (MisterX421138) [inline src/Twee/HomeRoutine/Good morning.twee]:
```suggestion
				@@.hero;Why did I touch myself.@@ you groan as you squirm from the post-orgasm stimulation.
```

File: src/Twee/HomeRoutine/Good morning.twee [hunk 5]
```diff
@@ -620,12 +620,21 @@ Noticing your worried face at her angry outburst, she quickly reassures you.
...
+				
+				You decide to not masturbate in the morning, no matter what.
```
Comment (MisterX421138) [inline src/Twee/HomeRoutine/Good morning.twee]:
```suggestion
				Unable to handle the uncomfortable feeling, you resolve to not masturbate in the morning, no matter what.
```

**Review by MisterX421138 — CHANGES_REQUESTED** [review]
Mostly text edits.  If you're good with them, just give them your approval.

**Comment by prndev** [comment]
Thank you for looking into it. Thank you for the corrections, they are looking good. This is probably the most comfortable way I ever improved my English skills. :)

**Comment by prndev** [comment]
As this pull request did not make it into 1.1000, I am closing it for now. I want to try again, opening a pull-request against the main repository next month.

**Pull request closed without merging**